### ISSUE TITLE
Is there any possible that IINA support LRC

### ISSUE THREAD

**Issue by kuangshenyan** [iina/iina#2658]
now iina can play music, but it doesn't support lrc, so, is there any possible to support local lrc or lrc from a url ?

**Comment by DroidFreak32** [comment]
I second this as well.
Coming from PotPlayer on windows, I really miss its .lrc file support.

**Comment by xjbeta** [comment]
MPV / IINA can load lrc file as subtitle.
Menu - Subtitles - Load External Subtitle...
Select .lrc file.

![891FDD63-56A6-4756-819D-2B231BD3FE89](https://user-images.githubusercontent.com/11794321/70623079-3d567f00-1c58-11ea-8da2-c9eee85647a9.png)

**Comment by kuangshenyan** [comment]
I want to have a mechanism where I can switch to music player mode, after which IINA can automatically load IRC files from a specified directory, and then match the current song and display.

**Comment by kuangshenyan** [comment]
> MPV / IINA can load lrc file as subtitle.
> Menu - Subtitles - Load External Subtitle...
> Select .lrc file.
> 
> ![891FDD63-56A6-4756-819D-2B231BD3FE89](https://user-images.githubusercontent.com/11794321/70623079-3d567f00-1c58-11ea-8da2-c9eee85647a9.png)

**Comment by lhc70000** [comment]
I think it's possible to implement this, at least as a plugin.

Please wait a few more days before I can resume the development.

**Comment by kuangshenyan** [comment]
thank you very much.

**Comment by francoestrubia** [comment]
I just commented this on other threat. 
IINA actually renders lrc files (don't remember filepaths).
Is just matter of make it look for lrc files, as do for regular subs. 
(and add it to music mode render pipeline).
LRC files has allways the same audiofile name.

**Comment by DroidFreak32** [comment]
I agree with @francoestrubia , IINA plays the .lrc file just fine.
The only missing thing is IINA does not proactively search for a .lrc file with the same name and play it automatically.

**Comment by francoestrubia** [comment]
Exactly. Proactively search can be such a task to write, but here is already coded.
Is all working, except you need to manually load the file.

**Comment by TianyiShi2001** [comment]
The above PR adds `.lrc` to the array of supported subtitle file extensions. After merging it, an additional change will be required: currently subtitle autoload only searches subtitles for videos, not audios. Specifically, changes will be required near

https://github.com/iina/iina/blob/0bc3bf07680f9d8b81ddd7f66067f0cd4b376134/iina/AutoFileMatcher.swift#L193-L195

to allow search subtitles also for audio files.

I'm not familiar with Swift so I won't be make this change soon. If anyone is familiar with Swift please do it!

**Comment by James-Harden-13-R** [comment]
![image](https://user-images.githubusercontent.com/56295468/114690664-81282f80-9d49-11eb-8640-541fbe4d9b3f.png)

I operated as mentioned above, but couldn't see the lyrics that I imported. So what's wrong with this?
Thanks!

**Comment by Rivers47** [comment]
> ![image](https://user-images.githubusercontent.com/56295468/114690664-81282f80-9d49-11eb-8640-541fbe4d9b3f.png)
> 
> I operated as mentioned above, but couldn't see the lyrics that I imported. I wonder what's wrong with this. Thanks!

It only works when the file contains an video stream (i.e. when it contains a cover art)
Otherwise it seems iina can't render the subtitle only

**Comment by IlIllIl** [comment]
Can iina support reading embedded LRC?

**Comment by jjm2473** [comment]
@guohui-ma Audio files without album cover cannot display .lrc lyrics. https://github.com/iina/iina/issues/5618
